src/profile/ProfilePage.jsx

```diff
@@ -362,6 +365,9 @@ const ProfilePage = ({ params }) => {
                     {...commonFormProps}
                   />
                   )}
+
+                  <ExtendedProfileFieldsSlot />
```

Comment:
I'm curious about the choice to put this above the `SocialLinks` block. I'd think having all the "out of the box" fields come before any additional ones would make more sense, but if there's specific reasoning for not putting this at the end I'm all ears!